src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final String DEFAULT_COLOR = "white";
+    private static final int DEFAULT_RADIUS = 10;
+
+    public Figure getRandomFigure() {
+        int figurType = RANDOM.nextInt(5);
+        String color = new ColorSupplier().getRandomColor();
+        switch (figurType) {
+            case 0:
+                return new Circle(color, RANDOM.nextInt(10) + 1);
+            case 1:
+                return new Square(color, RANDOM.nextInt(10) + 1);
+            case 2:
+                return new Rectangle(color, RANDOM.nextInt(10) + 1, RANDOM.nextInt(10) + 1);
+            case 3:
+                return new RightTriangle(color, RANDOM.nextInt(10) + 1,
+                        RANDOM.nextInt(10) + 1, RANDOM.nextInt(10) + 1);
+            case 4:
+                return new IsoscelesTrapezoid(color, RANDOM.nextInt(10) + 1,
+                        RANDOM.nextInt(10) + 1, RANDOM.nextInt(10) + 1);
+            default:
+                return getDefaultFigure();
+        }
+    }

```

Comment:
All magic numbers in your code should be constants. For example, the number '5' used in 'nextInt(5)' should be a named constant to indicate the number of figure types.